spec/controllers/forgot_passwords_controller_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe ForgotPasswordsController, :type => :controller do 
```

Comment:
Did you know you don't have to do that ':type => controller' ? RSpec should infer the type based on the directory name.